modules/ignite-cli-2/src/test/java/org/apache/ignite/cli/commands/cliconfig/CliConfigSetSubCommandTest.java

```diff
@@ -0,0 +1,75 @@
+package org.apache.ignite.cli.commands.cliconfig;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import jakarta.inject.Inject;
+import org.apache.ignite.cli.commands.CliCommandTestBase;
+import org.apache.ignite.cli.config.Config;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class CliConfigSetSubCommandTest extends CliCommandTestBase {
+
+    @Inject
+    Config config;
+
+    @BeforeEach
+    void setUp() {
+        setUp(CliConfigSetSubCommand.class);
+    }
+
+    @Test
+    @DisplayName("Need at least one parameter")
+    void noKey() {
+        // When executed without arguments
+        execute();
+
+        // Then
+        assertThat(err.toString()).contains("Missing required parameter");
+        // And
+        assertThat(out.toString()).isEmpty();
+    }
+
+    @Test
+    @DisplayName("Missing value")
+    void singleKey() {
+        // When executed with key
+        execute("ignite.cluster-url");
+
+        // Then
+        assertThat(err.toString()).contains("should be in KEY=VALUE format");
+        // And
+        assertThat(out.toString()).isEmpty();
+    }
+
+    @Test
+    @DisplayName("Sets single value")
+    void singleKeySet() {
+        // When executed with key
+        execute("ignite.cluster-url=test");
+
+        // Then
+        assertThat(out.toString()).isEqualTo("OK" + System.lineSeparator());
+        // And
+        assertThat(err.toString()).isEmpty();
+        // And
+        assertThat(config.getProperty("ignite.cluster-url")).isEqualTo("test");
+    }
+
+    @Test
+    @DisplayName("Sets multiple values")
```

Comment:
Why then we do not allow to read multiple values?

Reply:
Getting multiple keys will change the output format to be in form "key=value", I'm not sure if it's really useful. Probably we should disallow setting multiple keys as well.